net.retrodeck.retrodeck.yml

```diff
@@ -232,7 +232,7 @@ modules:
     sources:
       - type: git
         url: https://gitlab.com/es-de/emulationstation-de
-        branch: 18b2ee412e1bb3ea35ffd6e680ebf1dacda8e0ff
+        branch: f8d782f1f3b1fba8a119b793edd01f37e0258953
```

Comment:
Maybe this was unintentional?

Reply:
Yes, it seems to have been an artifact of the changing codebase. I have changed this PR to point at 0.6.0b and it now looks as intended.